src/main/webapp/app/shared/category-selector/category-selector.component.ts

```diff
@@ -52,22 +50,22 @@ export class CategorySelectorComponent implements OnInit, OnChanges {
         );
     }
 
-    private categoriesAsStringArray() {
```

Comment:
I don't like setting all methods to public. You don't need to test every method on its own to test all lines and branches.
Please revert the changes to the access modifiers and focus on testing the already public methods. You can think about what scenarios reach the respective part of the code you are trying to test and use that as a test case.

Keep in mind, you can also modify `test:diff` by adding a `--coverage` parameter to it and you can quickly check what lines of your component are not yet covered. Just don't commit the change, we don't need it on default.